runelite-client/src/main/java/net/runelite/client/plugins/prayer/PrayerPlugin.java

```diff
@@ -296,4 +308,50 @@ private void removeOverheadsIndicators()
 		infoBoxManager.removeIf(entry -> entry instanceof PrayerCounter
 			&& ((PrayerCounter) entry).getPrayerType().isOverhead());
 	}
+
+	private void setPrayerOrbText(String text)
+	{
+		Widget prayerOrbText = client.getWidget(WidgetInfo.MINIMAP_PRAYER_ORB_TEXT);
+		if (prayerOrbText != null)
+		{
+			prayerOrbText.setText(text);
+		}
+	}
+
+	private double getPrayerDrainRate(Client client)
```

Comment:
This can be `static`
```suggestion
	private static double getPrayerDrainRate(Client client)
```

Reply:
Didn't think of that, thanks!